docs/mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: PathforaJS SDK
-nav:
+pages:
```

Comment:
this change seems unrelated?

Reply:
I noticed this yesterday when deploying docs that changing this messed up the navigation so I'm changing it back, but yes it is unrelated to this PR.